staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -459,29 +459,31 @@ func (s *sharedIndexInformer) Run(stopCh <-chan struct{}) {
 		klog.Warningf("The sharedIndexInformer has started, run more than once is not allowed")
 		return
 	}
-	fifo := NewDeltaFIFOWithOptions(DeltaFIFOOptions{
-		KnownObjects:          s.indexer,
-		EmitDeltaTypeReplaced: true,
-		Transformer:           s.transform,
-	})
 
 	cfg := &Config{
-		Queue:             fifo,
 		ListerWatcher:     s.listerWatcher,
 		ObjectType:        s.objectType,
 		ObjectDescription: s.objectDescription,
 		FullResyncPeriod:  s.resyncCheckPeriod,
 		RetryOnError:      false,
 		ShouldResync:      s.processor.shouldResync,
 
-		Process:           s.HandleDeltas,
-		WatchErrorHandler: s.watchErrorHandler,
+		Process: s.HandleDeltas,
```

Comment:
Can you run `got fmt` on this? The formatting is odd over here.

Reply:
This is formatted, CI checks it. Its from the Line on L470 which was already present

Reply:
Hmm. It looks odd on GitHub UI. Maybe just a GitHub thing. All good. Thanks